src/Component/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,62 @@
+import { useState } from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  visibleTodos: Todo[];
+  isLoading: boolean;
+};
+
+export const TodoList: React.FC<Props> = ({ visibleTodos, isLoading }) => {
+  const [isEditing] = useState(false);
```

Comment:
You define a state variable 'isEditing' but never change it. It's always false. You should implement a way to change this state to allow users to edit todo items. But you can create a variable for this